### PR TITLE
Test Amin UsersController

### PR DESCRIPTION
**Pull request by anhtu268**
=> Title:

=> Redmine:

=> Attachment File:

### CODE REVIEW

**Comment by anhtu268** [comment]
ready

**Comment by anhtu268** [comment]
ready

File: app/models/reservation_detail.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class ReservationDetail < ApplicationRecord
+  class ReservationDetail < ApplicationRecord
```
Comment (huuphuc93) [inline app/models/reservation_detail.rb]: indent 2 spaces

**Comment by anhtu268** [comment]
ready

**Pull request merged**